src/icons/icons.qrc

```diff
@@ -308,13 +307,15 @@
         <file>qbt-theme/preferences-other.svg</file>
         <file>qbt-theme/preferences-system-network.svg</file>
         <file>qbt-theme/preferences-web-browser-cookies.svg</file>
+        <file>qbt-theme/reannounce.svg</file>
```

Comment:
Any files deleted (gear and gear32) must also be removed from this file, otherwise the compiler still looks for them.

Reply:
Thanks. I forgot about that file. Made necessary changes.

Reply:
are you sure they are not used in WebUI?